x-pack/legacy/plugins/maps/public/layers/sources/es_search_source/es_search_source.js

```diff
@@ -367,23 +370,35 @@ export class ESSearchSource extends AbstractESSource {
           defaultMessage: `Results limited to most recent {topHitsSize} documents per entity.`,
```

Comment:
Can you add the `iInCircle` icon to the beginning of this line so that users can correlate why there's an icon?

Reply:
We decided that top-hits actually selects fewer features by design, and it is configured by the user. So we wouldn't show the info-button here.